buildpack.md

```diff
@@ -461,7 +475,17 @@ For each buildpack in the group, the lifecycle
 
 After analysis, the lifecycle MUST proceed to the build phase.
 
-## Phase #3: Build
+## Phase #3: Generation (optional)
+
+### Purpose
+
+The purpose of the generation phase is to generate Dockerfiles that can be used to select the runtime base image.
```

Comment:
Is "define the runtime image" maybe better? It leaves the door open to mutations caused by the Dockerfile. "select" feels too narrow.